book/specifications/kimchi/template.md

```diff
@@ -269,17 +269,23 @@ TODO: for each gate describe how to create it?
 #### Foreign Field Multiplication
 
 {sections.foreign_field_mul}
+
 #### Xor
 
 {sections.xor16}
 
-#### Not
+##### Not
 
 {sections.not_gadget}
+
 ##### And
```

Comment:
looks H4 instead of H5 is needed here as well

Reply:
Oh, I intentionally wrote a fifth level as an attempt to show that none of And or Not are actual gate types, but rather gadgets reusing Xor in some manner. But I can move it to H4 if it makes more sense.

Reply:
ah no, that makes sense

Reply:
but maybe they should be moved to a gadget section then?